pkg/util/util.go

```diff
@@ -1544,29 +1544,33 @@ func GetEnvWithDefault(key string, defaultval string) string {
 	return val
 }
 
+func IsInvalidKubeConfigError(err error) bool {
+	return strings.Contains(err.Error(), "invalid configuration")
+}
+
 //IsValidKubeConfigPath checks if specified `KUBECONFIG` value is a valid file i.e the path should exist and be a file
 //if env `KUBECONFIG` is set then that is used or default `KUBECONFIG` is checked
-func IsValidKubeConfigPath() bool {
+func IsValidKubeConfigPath() error {
 	v := os.Getenv("KUBECONFIG")
```

Comment:
Can you rename `v` to something that makes more sense?

Reply:
updating